# Mangiola-Lab-Open-Positions
This repository lists the current open positions in the Mangiola Laboratory 

The Stefano Mangiola Research Group at SAiGENCI (Adelaide) with co-appointment at WEHI (Melbourne) is at the forefront of computational immunogenomics, employing machine-learning techniques to advance cancer diagnosis and treatment. Our interdisciplinary and international team collaborates extensively with immunology, cancer biology and machine learning experts, leveraging state-of-the-art experimental data to unravel biological processes and novel biomedical applications. 

# \#1 Postdoc level A computational biology: Spatial and single-cell data analysis for cancer immunogenomics![image](https://github.com/user-attachments/assets/7d36bc62-a74f-409f-ba35-697894e2ae98)

APPLY: [https://careers.adelaide.edu.au/cw/en/job/514184/spatial-singlecell-computational-immunogenomics-for-cancer-diagnosis](https://careers.adelaide.edu.au/cw/en/job/514184/postdoc-level-a-computational-biology-spatial-and-singlecell-data-analysis-for-cancer-immunogenomics)

**Grant-Funded Researcher (Level A)**

Work type: Fixed term - Full-time 2 years

(Level A, MSc/PhD) $75,888 to $105,040 per annum plus an employer contribution of 17% superannuation.

Full application: https://careers.adelaide.edu.au/cw/en/job/513184/grantfunded-researcher-a

The position is for a postdoctoral fellow in spatial and single-cell computational immunogenomics for cancer diagnosis. This position will advance cancer diagnosis by analysing immune responses and treatment efficacy through data-driven research and a multi-omic approach. Exploring the dynamics of circulating immune cells in blood reveals crucial insights into the immune system's response, its efficacy against metastatic cancer, and its reaction to treatments. Profiling peripheral blood immune single cells and circulating cytokines, we uncovered patterns of immune cell communication and composition linked to metastatic progression (Mangiola et al. 2023).  

<img src="image_position_immunodiagnosis.png?raw=true" width="800">

This exciting position aims to expand this research to encompass the local tumour microenvironment and apply this approach to identify individual immune characteristics influencing immunotherapy success. We aim to employ a multiomic approach, analysing extensive patient data across all stages of disease progression. This research will utilise state-of-the-art facilities, including 10x Xenium, 10x CITE-seq, Milliplex and proteomics. 

We're searching for a dynamic individual for the Computational Biologist/Bioinformatician/Biostatistician role. Managing personal research funds is a possibility depending on the appointment level and the candidate's experience.

The circulating immune cell landscape stratifies metastatic burden in breast cancer patients
S Mangiola, R Brown, J Berthelet, S Guleria, C Liyanage, S Ostrouska, J Wilcox, M Merdas, PF Larsen, C Bell, J Schroder, L Mielke, J Mariadason, S Chang-Hao Tsao, Y Chen, VK Yadav, RL Anderson, S Vodala, D Merino, A Behren, B Yeo, AT Papenfuss, B Pal
bioRxiv 2023.11.01.565223; doi: https://doi.org/10.1101/2023.11.01.565223

A multi-organ map of the human immune system across age, sex and ethnicity
S Mangiola, M Milton, N Ranathunga, CSN Li-Wai-Suen, A Odainic, E Yang, W Hutchison, A Garnham, J Iskander, B Pal, V Yadav, JFJ Rossello, VJ Carey, M Morgan, S Bedoui, A Kallies, AT Papenfuss
bioRxiv 2023.06.08.542671; doi: https://doi.org/10.1101/2023.06.08.542671

To be successful you will need: (max 5B)
•	Explicitly address each selection criteria 
•	Attach a CV to the application including at least the following subsections: Education, Publication, Awards, GitHub profile.
•	PhD in Computational Biology, Bioinformatics, Computer Science, or a closely related field with a strong focus on machine learning and deep learning applications.
•	Demonstrable experience in analysing large-scale single-cell genomic data.
•	Proficiency in programming languages commonly used in computational biology and data science, such as Python and R, with the ability to handle complex data analysis tasks. Please provide publicly available examples.
•	A strong record of research, evidenced by publications in peer-reviewed journals or presentations at significant conferences, particularly in areas related to AI, language modelling, computational biology, bioinformatics, or immunogenomics.
Desirable:
•	2+ postdoctoral years of experience 
•	Extensive experience in handling, processing, and interpreting large-scale biological datasets, including single-cell RNA-Seq data.
•	Familiarity with immunogenomics, including the understanding of immune cell types, pathways, and mechanisms, especially in the context of cancer.

![image](https://github.com/user-attachments/assets/e46cd960-a10c-47b3-9ef9-ac55cfdc99c2)

# \#2 Postdoc level A computational biology: Multiomics pipeline development in R and AI/Machine-learning methods development

APPLY: [https://careers.adelaide.edu.au/cw/en/job/514194/grantfunded-researcher-a](https://careers.adelaide.edu.au/cw/en/job/514194/postdoc-level-a-computational-biology)

**Grant-Funded Researcher (Level A)**

Work type: Fixed term - Full-time 2 years

(Level A, MSc/PhD) $75,888 to $105,040 per annum plus an employer contribution of 17% superannuation.

The position is for a postdoctoral or research-assistant fellow expert in large-scale data analysis and advanced R programming. High-throughput single-cell and multiomic data are revolutionising medical research. There is a need to develop analysis methods and pipelines that scale to tens of millions of cells. Tidyomics (Nature Methods 2024) is an R software ecosystem that enhances the analysis and visualisation of high-dimensional omics data, applying the principles of tidy data analysis, a de facto standard in data science. Given its large adoption, we propose to improve the documentation, robustness, and interoperability of the Tidyomics ecosystem and extend it to spatial profiling technologies.

<img width="288" alt="image" src="https://github.com/user-attachments/assets/67a62195-b943-4daa-954f-3384eaeaed1f">

Tidyomics packages enable computational biologists to employ a user-friendly grammar to manipulate popular data containers across omics (genomics, transcriptomics, cytometry) and platforms (Bioconductor, Seurat). Tidyomics aggregates a growing user base and developer community, forming an international network that spans five continents. 
This exciting position, beyond developing tidyomics, also aims to develop high-performance single-cell and spatial analysis pipelines in R for high-performance and cloud computing. These pipelines will serve our popular CuratedAtlasQuery database, which deploys the curated and annotated single-cell universe to researchers and institutions. Using these pipelines, you will model single-cell data at scale to extract meaningful knowledge about cancer and the immune system. 
This role will be responsible for applying large-language models and other machine-learning tools to large-scale single-cell and spatial public data.
Key responsibilities: Develop and enhance the tidyomics ecosystem (GitHub: tidyomics), write documentation and create and deliver workshops nationally and internationally. Enhance the pipeline HPCell (GitHub: MangiolaLaboratory/HPCell) and the single-cell knowledge base CuartedAtlasQuery. Analyse large-scale single-cell and spatial data using AI and more standard tools in the context of cancer immunology.

To be successful you will need: (max 5B)
•	Explicitly address each selection criteria 
•	Attach a CV to the application including at least the following subsections: Education, Publication, Awards, GitHub profile.
•	MSc/PhD in Computational Biology, Bioinformatics, Computer Science, or a closely related field with a strong focus on machine learning and deep learning applications.
•	Proficiency R, with the ability to handle complex data analysis tasks. Please provide publicly available examples.
•	A record of research, evidenced by publications in peer-reviewed journals or presentations at significant conferences, particularly in areas related to AI, language modelling, computational biology, bioinformatics, or immunogenomics.

Desirable:
•	Demonstrable experience in analysing large-scale single-cell genomic data.
•	Extensive experience in handling, processing, and interpreting large-scale biological datasets, including single-cell RNA-Seq data.
•	Familiarity and proficiency with deep-learning libraries, and large-language models
•	Knowledge of Python and bash.
•	Familiarity with immunogenomics, including the understanding of immune cell types, pathways, and mechanisms, especially in the context of cancer.


