# Mangiola-Lab-Open-Positions
This repository lists the current open positions in the Mangiola Laboratory 

The Stefano Mangiola Research Group at SAiGENCI (Adelaide) with co-appointment at WEHI (Melbourne) is at the forefront of computational immunogenomics, employing machine-learning techniques to advance cancer diagnosis and treatment. Our interdisciplinary and international team collaborates extensively with immunology, cancer biology and machine learning experts, leveraging state-of-the-art experimental data to unravel biological processes and novel biomedical applications. 

# \#1 Postdoc level A in Bioinformatics: Innovating Computational Tools for the Cell Data Universe

APPLY: https://careers.adelaide.edu.au/cw/en/job/516018/postdoc-level-a-in-bioinformatics-innovating-computational-tools-for-the-cell-data-universe

**Grant-Funded Researcher (Level A)**

(Level A) $78,544 to $105,611 per annum plus an employer contribution of 17% superannuation applies.

Fixed term, full time position available for 24 months.

The Mangiola Group at SAiGENCI, seeks a highly motivated Postdoctoral Fellow to lead the development of advanced bioinformatic tools and methods for analysing and harmonising large-scale single-cell and spatial cell data. This position offers an exciting opportunity to shape the future of computational approaches in precision oncology by driving innovation in data science.

About the Position

This role is focused on creating state-of-the-art computational frameworks for large-scale data manipulation and analysis, prioritising the development of tools that address the complexity of single-cell and spatial data. The successful candidate will contribute to building robust, scalable software solutions that enhance reproducibility, usability, and insight generation for high-dimensional datasets. Those include tidyomics (Hutchison et al., 2024), HPCell, and more.  The development will be done primarily in R and possibly in Python. Our solutions might include the implementation of deep-learning models on our data resources.

Key responsibilities include:

- Designing and implementing novel bioinformatic tools and interfaces, primarily in R with extensions in Python, to handle the challenges of multiomic data integration.
- Developing and expanding tidyomics (Hutchison et al., 2024), a modular software ecosystem for data manipulation and analysis, ensuring it meets the needs of complex datasets in biomedical research.
- Creating and optimising pipelines to support harmonisation, scalability, and computational efficiency for analysing the cell data universe.
- Incorporating cutting-edge methods, including aspects of artificial intelligence, to complement bioinformatic tools.

To be successful you will need:

- PhD in Computational Biology, Bioinformatics, Computer Science, or a closely related field.
- Demonstrable experience in analysing large-scale single-cell genomic data.
- Proficiency in programming languages commonly used in computational biology and data science, such as R and possibly Python, with the ability to handle complex data analysis tasks. Please provide publicly available examples.
- A strong record of research, evidenced by publications in peer-reviewed journals or presentations at significant conferences, particularly in areas related to bioinformatics, computational biology, bioinformatics, or immunogenomics.

Desirable:

- Familiarity with R tidyverse
- Extensive experience in handling, processing, and interpreting large-scale biological datasets, including single-cell RNA-Seq data.
- Experience in training foundational models of cell biology
- Familiarity with immunogenomics, including the understanding of immune cell types, pathways, and mechanisms, especially in the context of cancer.
 
