# Mangiola-Lab-Open-Positions
This repository lists the current open positions in the Mangiola Laboratory 

The Stefano Mangiola Research Group at SAiGENCI (Adelaide) with co-appointment at WEHI (Melbourne) is at the forefront of computational immunogenomics, employing machine-learning techniques to advance cancer diagnosis and treatment. Our interdisciplinary and international team collaborates extensively with immunology, cancer biology and machine learning experts, leveraging state-of-the-art experimental data to unravel biological processes and novel biomedical applications. 

# \#1 Position: AI large-language models of single-cell data for cancer immunogenomics

APPLY: https://careers.adelaide.edu.au/cw/en/job/514185/ai-largelanguage-models-of-singlecell-data-for-cancer-immunogenomics

Work type: Fixed term - Full-time

(Level B) $107,276 to $126,894 per annum plus an employer contribution of 17% superannuation applies.

Develop AI-driven digital tissue models for precision oncology, harnessing a multidisciplinary approach to tailor immunotherapy across diverse demographic profiles.

The South Australian Immunogenomics Cancer Institute (SAiGENCI) is embarking on a ground-breaking project to create a digital tissue AI model for precision oncology. This role offers an exciting opportunity to be part of a multidisciplinary team that combines expertise in AI, bioinformatics, clinical oncology, and cancer immunology. As a key team member, you will contribute to advancing our understanding and implementation of immunotherapy tailored to demographic specificities such as age and sex. The role involves the development and refinement of AI-driven tissue models (e.g. large-language models) and the analysis of extensive single-cell data across a wide demographic spectrum.

This position will allow you to explore and develop new methodologies that integrate detailed cellular and molecular data to enhance the precision of established therapies and uncover new treatment avenues. We are looking for experienced individuals who are passionate about using their skills to make a tangible impact in medical research. You will be part of a team that values collaboration, innovation, and the pursuit of excellence. If you are driven to be at the cutting edge of technology and healthcare, we would like to hear from you.

# \#2 Position: Spatial and single-cell computational immunogenomics for cancer diagnosis

APPLY: https://careers.adelaide.edu.au/cw/en/job/514184/spatial-singlecell-computational-immunogenomics-for-cancer-diagnosis

**Grant-Funded Researcher (Level A)**

Work type: Fixed term - Full-time 2 years

(Level A, MSc/PhD) $75,888 to $102,040 per annum plus an employer contribution of 17% superannuation.

Full application: https://careers.adelaide.edu.au/cw/en/job/513184/grantfunded-researcher-a

The position is for a postdoctoral fellow in spatial and single-cell computational immunogenomics for cancer diagnosis. This position will advance cancer diagnosis by analysing immune responses and treatment efficacy through data-driven research and a multi-omic approach. Exploring the dynamics of circulating immune cells in blood reveals crucial insights into the immune system's response, its efficacy against metastatic cancer, and its reaction to treatments. Profiling peripheral blood immune single cells and circulating cytokines, we uncovered patterns of immune cell communication and composition linked to metastatic progression (Mangiola et al. 2023).  

<img src="image_position_immunodiagnosis.png?raw=true" width="800">

This exciting position aims to expand this research to encompass the local tumour microenvironment and apply this approach to identify individual immune characteristics influencing immunotherapy success. We aim to employ a multiomic approach, analysing extensive patient data across all stages of disease progression. This research will utilise state-of-the-art facilities, including 10x Xenium, 10x CITE-seq, Milliplex and proteomics. 

We're searching for a dynamic individual for the Computational Biologist/Bioinformatician/Biostatistician role. Managing personal research funds is a possibility depending on the appointment level and the candidate's experience.

The circulating immune cell landscape stratifies metastatic burden in breast cancer patients
S Mangiola, R Brown, J Berthelet, S Guleria, C Liyanage, S Ostrouska, J Wilcox, M Merdas, PF Larsen, C Bell, J Schroder, L Mielke, J Mariadason, S Chang-Hao Tsao, Y Chen, VK Yadav, RL Anderson, S Vodala, D Merino, A Behren, B Yeo, AT Papenfuss, B Pal
bioRxiv 2023.11.01.565223; doi: https://doi.org/10.1101/2023.11.01.565223

A multi-organ map of the human immune system across age, sex and ethnicity
S Mangiola, M Milton, N Ranathunga, CSN Li-Wai-Suen, A Odainic, E Yang, W Hutchison, A Garnham, J Iskander, B Pal, V Yadav, JFJ Rossello, VJ Carey, M Morgan, S Bedoui, A Kallies, AT Papenfuss
bioRxiv 2023.06.08.542671; doi: https://doi.org/10.1101/2023.06.08.542671

# \#3 Position: tidyomics and HPC software engineer

APPLY: https://careers.adelaide.edu.au/cw/en/job/514194/grantfunded-researcher-a

**Grant-Funded Researcher (Level A)**

Work type: Fixed term - Full-time 2 years

(Level A, MSc/PhD) $75,888 to $102,040 per annum plus an employer contribution of 17% superannuation.

R software engineer for tidyomics ecosystem and single-cell and spatial omic pipelines

Develop the tidyomics ecosystem and scalable data analysis pipelines in R

The position is for a postdoctoral or research-assistant fellow in advanced R programming. Tidyomics (Nature Methods 2024) is an R software ecosystem that enhances the analysis and visualisation of high-dimensional omics data, applying the principles of tidy data analysis, a de facto standard in data science. Given its large adoption, we propose to improve the documentation, robustness, and interoperability of the Tidyomics ecosystem and extend it to spatial profiling technologies.

This exciting position also aims to develop high-performance single-cell and spatial analysis pipelines in R, for high-performance and cloud computing. These pipelines will serve our popular CuratedAtlasQuery database, which deploys the curated and annotated single-cell universe to researchers and institutions.

Key responsibilities: Develop and enhance the tidyomics ecosystem (GitHub: tidyomics), write documentation and create and deliver workshops nationally and internationally. Enhance the pipeline HPCell (GitHub: MangiolaLaboratory/HPCell) and the single-cell knowledge base CuartedAtlasQuery.
